src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs

```diff
@@ -115,7 +115,7 @@ public static IResourceBuilder<RedisResource> WithDataVolume(this IResourceBuild
     /// Use <see cref="WithPersistence(IResourceBuilder{RedisResource}, TimeSpan?, long)"/> to adjust Redis persistence configuration, e.g.:
     /// <code lang="csharp">
     /// var cache = builder.AddRedis("cache")
-    ///                    .WithDataBindMount()
+    ///                    .WithDataBindMount("myredisdata")
```

Comment:
We should do this for the other 2 places where this is copied.

https://github.com/dotnet/aspire/blob/2249f7d4d114784ed4f117a60b1b2963d971bd92/src/Aspire.Hosting.Valkey/ValkeyBuilderExtensions.cs#L105-L106

https://github.com/dotnet/aspire/blob/2249f7d4d114784ed4f117a60b1b2963d971bd92/src/Aspire.Hosting.Garnet/GarnetBuilderExtensions.cs#L98-L99